task.sql

```diff
@@ -9,9 +9,32 @@ CREATE TABLE Countries (
 ) ENGINE=InnoDB;
 
 -- Create a table for caching GeoIP data (Columns: ID, IP Range, CountryID)
+CREATE TABLE GeoIP (
+    ID INT AUTO_INCREMENT PRIMARY KEY,
+    IPRange VARCHAR (50),
+    CountryID INT
+) ENGINE=MEMORY;
 
 -- Create a table for storing product descriptions for different countries (Columns: ID, CountryID, ProductID, Description )
+CREATE TABLE ProductDescription (
+    ID INT AUTO_INCREMENT PRIMARY KEY,
+    Description VARCHAR (100),
+    ProductID INT,
+    CountryID INT,
+    FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE CASCADE,
```

Comment:
The `ProductDescription` table references a `Products` table that is not defined in the current script. Ensure that the `Products` table is created and defined before applying this foreign key constraint.